plexlibrary/recipe.py

```diff
@@ -21,6 +23,9 @@
 from recipes import RecipeParser
 from utils import Colors, add_years
 
+logging.basicConfig(format='%(levelname)s:%(message)s',
+                    level=logging.INFO)
```

Comment:
This should be moved, probably to `main()` so we can pass args, or to a constructor in `logs.py`. I'd like to be able to specify verbosity in the future. Timestamp would be nice too.